handler.go

```diff
@@ -504,10 +524,17 @@ func (s *handler) handle(ctx context.Context, req request, w func(func(io.Writer
 
 			log.Warnf("failed to setup channel in RPC call to '%s': %+v", req.Method, err)
 			stats.Record(ctx, metrics.RPCResponseError.M(1))
-			resp.Error = &respError{
+			respErr := &respError{
```

Comment:
Don't we need to do the same thing in the `rpcError` function in server.go ?

Reply:
Probably not as `rpcErr` is only used for RPC processing errors and not errors returned by the invocation of the function.